.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        spark: ["2.4.8","3.0.3","3.1.3","3.2.1"]
+        spark: ["3.1.3","3.2.1", "3.3.2", "3.4.2"]
```

Comment:
Let's update README too and mention, that spark-2 is not supported?

Reply:
Added Readme